yomigeek/mtracker-app
# #157988628 Implement Dashboard for User Requests

#### What does this PR do?
Implement Dashboard Page for User

#### Description of Task to be completed?
User should be able to view the list of requests logged

#### How should this be manually tested?
Go to the url https://mtrack-app.herokuapp.com/login.html
Enter the required credentials
You get a response if its successful or not
If successful the user dashboard page is displayed 

#### Any background context you want to provide?
None

####  What are the relevant pivotal tracker stories?
story type: feature
story id: 157988628 

#### What are the relevant Github Issues?
None

#### Questions:
None
